### ISSUE TITLE
Crossing detail

### ISSUE THREAD

**Issue by briaguya** [cityofaustin/ctxfloods#135]
https://projects.invisionapp.com/share/NMFFOHHWTBG#/screens/280010662

**Comment by briaguya** [comment]
<img width="449" alt="screen shot 2018-02-26 at 1 22 43 pm" src="https://user-images.githubusercontent.com/6384156/36690551-41692e0c-1af8-11e8-871e-a75d49c084d3.png">

**Comment by briaguya** [comment]
Status history is a separate task

**Comment by luqmaan** [comment]
@ohsk I was about to ask you how do you get to the crossing detail view/edit screen, but eventually figured it out.

![image](https://user-images.githubusercontent.com/1275831/37780191-3d3c7916-2dbc-11e8-9312-2eb7fbc78d62.png)

I had no idea that the crossing title is actually a link.

Is there any way we can make the edit link easier to find?

**Comment by ohsk** [comment]
@luqmaan yes, I do think we can emphasize it as a link more. A few quick ideas:
- Underline them by default, or give them an underline on hover
- Put an "Edit crossing" link underneath the crossing title, similar in typography to the link style on the far right that indicates the last user who edited that crossing

Do you have a sense of how frequently users will be / should be editing the crossing details?

**Comment by ohsk** [comment]
I guess it's also the way to view crossing status history and download the history, so should be pretty accessible. Maybe "Edit crossing" is the wrong text... "View details" or something similar might be more accurate?

**Comment by luqmaan** [comment]
My guess is that users won't need to edit crossings too often.

👍 for the underline. I'll update the CSS.

I slightly prefer "Edit crossing" since it was an action that I wanted to do but couldn't find. I'm not sure I would know to check the details view to edit the crossing.
